Library/Homebrew/software_spec.rb

```diff
@@ -405,17 +411,27 @@ def skip_relocation?
     cellar == :any_skip_relocation
   end
 
+  sig { params(tag: Symbol).returns(T::Boolean) }
   def tag?(tag)
     checksum_for(tag) ? true : false
   end
 
-  # Checksum methods in the DSL's bottle block optionally take
+  # Checksum methods in the DSL's bottle block take
   # a Hash, which indicates the platform the checksum applies on.
+  # Example bottle block syntax:
+  # bottle do
+  #  sha256 "69489ae397e4645..." => :big_sur, :cellar => :any_skip_relocation
+  #  sha256 "449de5ea35d0e94..." => :catalina, :cellar => :any
+  # end
+  # Example args:
+  # {"69489ae397e4645..."=> :big_sur, :cellar=>:any_skip_relocation}
   def sha256(val)
```

Comment:
```suggestion
  def sha256(hash)
```

might make the below more readable